spec/bolt/inventory_spec.rb

```diff
@@ -748,4 +748,37 @@ def common_data(transport)
         .to raise_error(Bolt::Inventory::ValidationError, /Unsupported version/)
     end
   end
+
+  context 'when using inventory show' do
+    let(:data) {
+      { 'nodes' =>
+                [{ 'name' => 'foo',
+                   'alias' => %w[bar baz],
+                   'config' => { 'ssh' => { 'disconnect-timeout' => 100 } },
+                   'facts' => { 'foo' => 'bar' } }] }
+    }
+
+    let(:inventory) { Bolt::Inventory.new(data) }
+    let(:target) { get_target(inventory, 'foo') }
+    let(:expected_data) {
+      { 'name' => 'foo',
+        'alias' => %w[bar baz],
+        'config' =>
+                          { 'transport' => 'ssh',
+                            'ssh' =>
+                            { 'connect-timeout' => 10,
+                              'tty' => false,
+                              'load-config' => true,
+                              'disconnect-timeout' => 100 } },
+        'vars' => {},
+        'facts' => { 'foo' => 'bar' },
+        'features' => [],
+        'plugin_hooks' =>
+                          { 'puppet_library' => { 'plugin' => 'puppet_agent', 'stop_service' => true } } }
+    }
```

Comment:
```suggestion
    let(:expected_data) {
      { 'name' => 'foo',
        'alias' => %w[bar baz],
        'config' => {
          'transport' => 'ssh',
          'ssh' => {
            'connect-timeout' => 10,
            'tty' => false,
            'load-config' => true,
            'disconnect-timeout' => 100
          }
        },
        'vars' => {},
        'facts' => { 'foo' => 'bar' },
        'features' => [],
        'plugin_hooks' => {
          'puppet_library' => {
            'plugin' => 'puppet_agent',
            'stop_service' => true
          }
        } }
    }
```

Probably best to match the formatting for other hashes in the file. This doesn't raise any rubocop errors for me.